include/grpcpp/impl/codegen/completion_queue.h

```diff
@@ -376,7 +376,7 @@ class ServerCompletionQueue : public CompletionQueue {
   ServerCompletionQueue() : polling_type_(GRPC_CQ_DEFAULT_POLLING) {}
 
  private:
-  /// \param is_frequently_polled Informs the GRPC library about whether the
+  /// \param polling_type Informs the GRPC library about whether the
```

Comment:
Given the enum types in `grpc_cq_polling_type`, I'm not sure this comment makes sense.

Reply:
@sreecha : can you fix this comment when you have a chance? It is outdated since it describes the old boolean way of doing things which is slightly different than the current enum way.